stl/inc/sstream

```diff
@@ -584,20 +649,17 @@ public:
         : _Mybase(_STD addressof(_Stringbuffer)), _Stringbuffer(_Str, _Mode | ios_base::in) {}
 #endif // _HAS_CXX20
 
-    basic_istringstream(basic_istringstream&& _Right) : _Mybase(_STD addressof(_Stringbuffer)) {
-        _Assign_rv(_STD move(_Right));
-    }
+    basic_istringstream(basic_istringstream&& _Right)
+        : _Mybase(_STD addressof(_Stringbuffer)),
+          _Stringbuffer(((void) _Mybase::swap(_Right), _STD move(_Right._Stringbuffer))) {}

```

Comment:
These `(void)` casts are unnecessary to prevent comma hijacking. `_Mybase` is `basic_istream` and we know its `swap` always returns `void`:

https://github.com/microsoft/STL/blob/442029c6fa37f1b6f9203357de09672d5704077c/stl/inc/istream#L59

Similarly for the others.